src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -140,6 +141,12 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Internal
 
+(schema/defn keylength :- schema/Int
+  "Returns either the specified keylength from the settings or the default keylength
+  if none were present in the settings"
+  [settings :- CaSettings]
+  (get settings :keylength utils/default-key-length))
```

Comment:
Someday I'd like to consolidate all of the definitions of the default values for various settings in a single spot, but I don't think we have that for this service yet... so, this seems good for now.